backend/svirt.pm

```diff
@@ -56,6 +56,13 @@ sub do_start_vm {
     return {};
 }
 
+sub is_shutdown {
+    my ($self) = @_;
+
+    my $instance = get_var('VIRSH_INSTANCE');
```

Comment:
there are like 4 places now that check the instance - and 3 of them have a default to 1. So you better extract that somewhere.